main_pullrequest.go

```diff
@@ -11,8 +12,32 @@ import (
 	clientgithub "github.com/mendersoftware/integration-test-runner/client/github"
 )
 
-func processGitHubPullRequest(ctx *gin.Context, pr *github.PullRequestEvent, githubClient clientgithub.Client, conf *config) error {
+func startPipeline(branchName string) bool {
```

Comment:
Suggest renaming to `shouldStartPipeline`. The original name makes it sound like you are starting the pipeline (and this is what I first thought when I read it out of context below).